app/panel/components/Settings.jsx

```diff
@@ -128,12 +138,13 @@ class Settings extends React.Component {
 	/**
 	 * Hide alert in 3 sec. after it has been shown.
 	 */
-	hideToast = debounce(function () {
+	hideToast = debounce(function() { // eslint-disable-line react/sort-comp
```

Comment:
@christophertino why do we need this `eslint-disable-line react/sort-comp` here?